lightning/kv/importer.go

```diff
@@ -102,43 +103,34 @@ func (importer *Importer) SwitchMode(ctx context.Context, mode sst.SwitchMode) e
 			Mode: mode,
 		},
 	}
-	timer := time.Now()
 
+	task := log.With(zap.Stringer("mode", mode)).Begin(zap.DebugLevel, "switch mode")
```

Comment:
why do we use `DebugLevel`?

Reply:
Originally there was no "switch mode start" log, so this was set to debug level to keep the log from showing by default.